addon/components/base/bs-form/element/control/input.js

```diff
@@ -32,7 +32,8 @@ export default Control.extend({
     'step',
     'form',
     'spellcheck',
-    'title'
+    'title',
+    'ariaDescribedby:aria-describedBy'
```

Comment:
I would put them in the parent until we know we have a child that doesn't need it. We can break it out if and when we add a special control. I think you can remove things from `attributeBindings` by creating a read-only `null` CP of that name.